app/controllers/user/birth_records_controller.rb

```diff
@@ -39,10 +37,8 @@ def add
   # Attempts to remove a record which is not attached or doesn't exist will be
   # silently ignored.
   def remove
-    AuditedOperationsService.remove_birth_record_from_user(
-      user: current_user,
-      birth_record_id: params.permit(:id)[:id].to_i
-    )
+    @birth_record = current_user.birth_records.find_by(params.permit(:id))
```

Comment:
how about something like
```
    @document = current_user.documents.find_by(params.permit(:id, :document_type))
```

Reply:
It's the birth records controller so they should always be birth records - would need to add a hidden `document_type` field